### ISSUE TITLE
False positive in nested return statements inside of getters

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_lint#1008]
### Lint Name

`getter-return`

### Code Snippet

```ts
class _Test {
  get foo() {
    const target = {
      asd() {
        asd;
        return;
      },
    };
    return target;
  }
}
```

### Expected Result

No lint error.

### Actual Result

```
(getter-return) Expected to return a value in 'foo'.
        return;
        ^^^^^^^
    at test.js:6:8

    hint: Return a value from the getter function
    help: for further information visit https://lint.deno.land/#getter-return

Found 1 problem
Checked 1 file
```

### Version

```
deno 1.18.2 (release, x86_64-unknown-linux-gnu)
v8 9.9.115.5
typescript 4.5.2
```
